common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -160,8 +160,12 @@ public class CommonValue {
                         + "\" : {\"type\": \"flat_object\"},\n"
                         + "      \""
                         + AbstractConnector.ACTIONS_FIELD
+                        + "\" : {\"type\": \"flat_object\"},\n"
+                        + "      \""
+                        + AbstractConnector.MODEL_INTERFACE_FIELD
                         + "\" : {\"type\": \"flat_object\"}\n";
```

Comment:
From my understanding, this interface field is for model, not only for connectors but also local models like text_embeddings etc. Why do we define it only in connector, but in model?

Reply:
Moved out to Model Index